README.md

```diff
@@ -1,41 +1,54 @@
-# nix-config
+# nix-config ![CI](https://github.com/lovesegfault/nix-config/workflows/CI/badge.svg)
 
-This repository holds both my NixOS configuration (`system/`) as well as my Home
-Manager configuration (`home/`), both work together to generate fully
-configured, consistent, systems to my liking.
+This repository holds both my NixOS configuration. It is fully reproducible
+(through [niv]) and position-independent, meaning there is no moving around of
+`configuration.nix` (see [switch]).
+
+For the configurations' entry points see the individual [systems], as well as
+[default.nix]. For adding users or overlays see [users](#users),
+[overlays](#overlays), respectively.
 
 ## structure
 ```
-home/
-├── combo/            # Meta-configurations (e.g. core, desktop, development)
-├── hardware/         # Hardware-specific configuration (e.g. DPI, resolution)
-├── machines/         # Configuration for actual hosts (selectively imports combos)
-├── modules/          # Composable modules (e.g. neovim, zsh, sway)
-└── pkgs/             # Personal packages (e.g. swaymenu, patched weechat)
-
-system/
-├── combo/            # Meta-configurations (e.g. core, desktop, development)
-├── hardware/         # Hardware-specific configuration (e.g. kernel modules, peripherals)
-├── machines/         # Configuration for actual hosts (selectively imports combos)
-├── modules/          # Composable modules (e.g. bumblebee, sudo, sway, xserver)
-└── pkgs/             # Personal packages (e.g. kernel patches)
-
-secrets/              # Secrets (API keys)
-
-misc/
-├── config.nix        # Example NixOS user configuration
-├── configuration.nix # Example NixOS configuration
-├── home.nix          # Example home-manager configuration
-└── hostnames         # List of hostnames to use
+.
+├── core         # Base-line configurations applicable to all machines
+├── default.nix  # Attr set of all machines plus arch-groups
```

Comment:
```suggestion
├── default.nix  # Attr set of all machines, with groups based on architecture
```
Took my brain a few seconds to make the leap from arch -> architecture (btw I use Arch). Minor nitpick, and totally unnecessary, but improves readability IMHO.